yarn-project/stdlib/src/abi/abi.ts

```diff
@@ -340,6 +340,10 @@ export interface ContractArtifact {
   /** The functions of the contract. */
   functions: FunctionArtifact[];
 
+  // TODO(MW): below is WIP
+  /** The public functions of the contract, excluding dispatch. */
+  publicFunctions: FunctionAbi[];
+
```

Comment:
Undecided whether to keep this property or just use `.functions` and leave the public bytecode field empty (unless it's `public_dispatch`). This will probably change!!